### DISCUSSION TITLE
Creating Cargets

### DISCUSSION THREAD

**Discussion by KleighKennnedy** [cssbattle/forum#380] [Q&A]
I am really enjoying CSS Battle, and thought that it might be fun to try and make a few targets. I have tried CodePen and other similar platforms, but I haven't found a way to make the targets easily (using the 800x600 rule). Any suggestions on platforms or other methods that I should try to make some targets?

**Comment by chinchang** [comment]
Hi @KleighKennnedy! I am not sure I got your use case here 🤔 Do you want to create fresh targets from scratch, different from what are available on CSSBattle?

**Reply by KleighKennnedy** [reply]
Yes, I would like to create fresh targets from scratch. It seems at though there is a new feature that allows you to do so, but in order to I would need to upload a photo, and I don't know what ways I could control the size of the photo or if there were any websites that catered to making CSS art

**Comment by Vincent-Viton** [comment]
That's an excellent question, KleighKennedy. I've had a similar query myself. Is there a way to submit suggestions for targets that other users can play? I have several ideas, but the only feature I've encountered that somewhat resembles this is the "private battles" option, which, unfortunately, comes with a hefty price tag, rendering it impractical and defeating the purpose as nobody will see the private targets.

Regarding your other question, I believe the optimal method for creating a target is through... HTML and CSS! Begin by rendering an 800x600 div and verify its appearance once reduced by zooming the browser to 50%. Adding a distinct box-shadow to the div, such as pink or any uncommon color, can help delineate its outline. Afterward, take a screenshot and paste it into an image editing software like GIMP. Carefully crop the image to precisely 800x600 pixels and export it as png.

But of course, this make sense only if the community can contribute.

**Reply by KleighKennnedy** [reply]
Thanks for the reply. I'll give that a go, hopefully CSSBattle would make a feature that would allow this to happen more cleanly :).
